modules/ROOT/pages/backup-restore/planning.adoc

```diff
@@ -184,18 +169,44 @@ They must be run directly on the databases that are associated with that Composi
 | {check-mark}
 |===
 
+
+[NOTE]
+====
+The Neo4j Admin commands `backup`, `restore`, `dump`, `load`, `copy`, and `check-consistency` are not supported for use on xref:database-administration/composite-databases/concepts.adoc[Composite databases].
+They must be run directly on the databases that are associated with that Composite database.
+====
+
+
+== Considerations for backing up and restoring databases in a cluster
+
+Backing up a database in a clustered environment is not essentially different from a standalone backup, apart from the fact that you must know which server in a cluster to connect to.
+Use `SHOW DATABASE <database>` to learn which servers are hosting the database you want to back up.
+See xref:clustering/monitoring/show-databases-monitoring.adoc#show-databases-monitoring-listing-single[Listing a single database] for more information.
+
+However, _restoring_ a database in a cluster is different since it is not known in advance how a database is going to be allocated to the servers in a cluster.
+This method relies on the seed already existing on one of the servers.
```

Comment:
I feel like it's not totally clear what 'this method' refers to, and I'm a little confused by the 'existing on one of the servers' now I look at it.

Perhaps something like:
> Restoring from the command line involves putting a copy of the database on disk on each server that will need it.
> That can be awkward to achieve.